src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
+
+    if (userInput.length > 0) {
+      const lat = userInput[0].lat.toString();
```

Comment:
you will likely want to add a comment here explaining what you are doing, as choosing the first user and ignoring the rest is non-obvious (but that is what we want to do for now)